CHANGELOG.md

```diff
@@ -8,6 +8,107 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| [Behavioral Twin: Health Indicators (HI) KIT](https://eclipse-tractusx.github.io/docs-kits/category/health-indicator-hi-kit)                                      |                                                       n/a                                                       |                   [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20HI%20Kit/Changelog%20Health%20Indicator%20Kit)                   |
+| [Behavioral Twin: Model and Data processing (MDP) KIT](https://eclipse-tractusx.github.io/docs-kits/category/model-based-development-and-data-processing-mdp-kit) |                                                       n/a                                                       | [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20MDP%20Kit/Changelog%20Model%20Based%20Development%20and%20Data%20Processing%20Kit) |
+| [Behavioral Twin: Remaining useful Life (RUL) KIT](https://eclipse-tractusx.github.io/docs-kits/category/remaining-useful-life-rul-kit)                           |                                                       n/a                                                       |               [0.1.1](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20RuL%20Kit/Changelog%20Remaining%20Useful%20Life%20Kit)               |
+| [Circularity KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/page-adoption-view)                                                           |                                                       n/a                                                       |                            [0.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/Changelog%20Circularity%20KIT)                             |
+| [Modular Production KIT](https://eclipse-tractusx.github.io/docs-kits/category/modular-production-kit)                                                            |                                                       n/a                                                       |                  [0.1.0](https://eclipse-tractusx.github.io/docs-kits/kits/Modular%20Production%20Kit/Modular%20Production%20Kit%20Changelog)                   |
+
+
```

Comment:
new category with table: "deprecated"
Behavioral Twin: Health Indicators (HI) KIT
Behavioral Twin: Model and Data processing (MDP) KIT
Behavioral Twin: Remaining useful Life (RUL) KIT
Certificate Management KIT

all of those with versions as mentioned in March (R24.03)

Reply:
done

Reply:
Wouldn't "removed" be more appropriate? Something that's deprecated is due for removal but still there. I think those mentioned above will actually be gone with R24-05.

Reply:
Aligned on "Removed"